moped-editor/src/components/GridTable/Filters.js

```diff
@@ -578,10 +579,14 @@ const Filters = ({
                             : option
                         }
                         onChange={(e, value) => {
-                          handleSearchValueChange(
-                            filterId,
-                            value[filterParameters[filterId].lookup_field]
-                          );
+                          if (value) {
+                            handleSearchValueChange(
+                              filterId,
+                              value[filterParameters[filterId]?.lookup_field]
+                            );
+                          } else {
+                            handleSearchValueChange(filterId, value);
+                          }
```

Comment:
this fixes an error i came across that happens if you select a value from an autocomplete and then use the x to remove it. 
<img width="1039" alt="Screenshot 2023-10-25 at 3 59 34 PM" src="https://github.com/cityofaustin/atd-moped/assets/12474808/0bc90c71-d117-43c6-9a12-a3dd696a4a14">

Reply:
would you add a comment in the `else` block about what's going on? no value here is just `""`?